### DISCUSSION TITLE
Arctos urls in dwc-A are http?

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4679] [General]
See conversation at https://github.com/globalbioticinteractions/globalbioticinteractions/issues/791#issuecomment-1120094653

**Comment by dustymc** [comment]
They are, and that's probably worthy of discussion.

At least at one time as a sort of best practice it was seen as proper (er sumthin) to use http URLs as GUIDs, and that's what ours started out as so I've never updated them. I've also probably not been consistent in that somewhere, and users probably just grab whatever's in their browser's URL bar when they need a GUID. Ideally we'd somehow be more consistent, practically I'm not so sure it matters - anyone looking for URLs has more or less been trained by reality to look for http and https at this point.

That's currently complicated by https://github.com/ArctosDB/arctos/issues/4377 - TACC is currently pushing to https, but not completely blocking http and in some circumstances the http page is served to users. I've got a request in, I'm not sure where they are with it.
